tests/conftest.py

```diff
@@ -1,8 +1,76 @@
 import pathlib
-
 import pytest
+import os
+
+from zarr.v3.store import LocalStore, StorePath, MemoryStore
+from zarr.v3.store.remote import RemoteStore
 
 
 @pytest.fixture(params=[str, pathlib.Path])
 def path_type(request):
     return request.param
+
+
+@pytest.fixture()
+def mock_s3(request):
```

Comment:
as you said in another comment, let's move the remote store setup / testing to another PR.

Reply:
removed in b28eaee2f328e8e9ee2aed9f51d14edf9d49630b